keras/layers/recurrent.py

```diff
@@ -151,17 +153,28 @@ class Recurrent(Layer):
     # Note on using dropout with TensorFlow
         When using the TensorFlow backend, specify a fixed batch size for your model
         following the notes on statefulness RNNs.
+
+    # References
+        - [Recurrent Batch Normalization](https://arxiv.org/abs/1603.09025)
     '''
     def __init__(self, weights=None,
                  return_sequences=False, go_backwards=False, stateful=False,
                  unroll=False, consume_less='cpu',
-                 input_dim=None, input_length=None, **kwargs):
+                 input_dim=None, input_length=None, batch_norm=False, batch_norm_gamma=0.1, **kwargs):
         self.return_sequences = return_sequences
         self.initial_weights = weights
         self.go_backwards = go_backwards
         self.stateful = stateful
         self.unroll = unroll
         self.consume_less = consume_less
+        self.batch_norm = batch_norm
+        self.batch_norm_gamma = batch_norm_gamma
+        if self.batch_norm:
+            self.beta_init = initializations.get('zero')
+            self.epsilon = 1e-6
+            self.uses_learning_phase = True
```

Comment:
You are setting this to True but don't seem to be using the learning phase in the code.